tests/scripts/check-generated-files.sh

```diff
@@ -129,6 +129,7 @@ check()
 check scripts/generate_errors.pl library/error.c
 check scripts/generate_query_config.pl programs/test/query_config.c
 check scripts/generate_driver_wrappers.py library/psa_crypto_driver_wrappers.h
+check scripts/generate_driver_wrappers.py  library/psa_crypto_driver_wrappers_no_static.c
```

Comment:
```suggestion
check scripts/generate_driver_wrappers.py library/psa_crypto_driver_wrappers.h library/psa_crypto_driver_wrappers_no_static.c
```

Reply:
That may break the regex search for those files in [code_style.py ](https://github.com/Mbed-TLS/mbedtls/blob/development/scripts/code_style.py#L45)

I will test and update accodingly

Reply:
Done, thanks for catching it

Reply:
https://github.com/Mbed-TLS/mbedtls/commit/55674084602cbaee01e472a3a02474b4b8f24130 still runs `scripts/generate_driver_wrappers.py` twice.

Reply:
I know but that that script is performing two tasks:

- Generates automatically generated files when invoked.
- Acts as the truth of the autogenerated files to ignore from the code-style checker.

As such [the regex pattern](https://github.com/Mbed-TLS/mbedtls/blob/development/scripts/code_style.py#L45-L46) strictly defines what I can do to update it.

Alternatively I could be updating the pattern and add the line like

`#check scripts/generate_driver_wrappers.py library/psa_crypto_driver_wrappers.h library/psa_crypto_driver_wrappers_no_static.c`

So it would not be executed twice, but that feels even more hacky. I think we can take the cost of a double generation for now.

Reply:
The regex pattern enforces the syntax `check PROGRAM FILE...` where elements are separated by one or more blank and can't contain certain special characters, for each generated file that `code_style.py` would look at (so, in practice, `.c` files but not `.data` files). Having multiple files generated by the same program is supported.